src/component/absensi/employee/Employee.vue

```diff
@@ -121,18 +122,18 @@
             text-align: center;
             border: 1px solid transparent;
             border-radius: 4px;
-            background-color: #b3dcff;
+            background-color: $button__light-blue;
             cursor: pointer;
             text-decoration: none;
             color:black;
             &:hover{
-                background-color: #94c2ee;
+                background-color: $button__light-blue-hover;
```

Comment:
Okay, I'm confused. I don't remember that our rule has this modifier like `-hover`, and how does we differentiate the dash (`-`) from `light-blue` and `lalala-hover`